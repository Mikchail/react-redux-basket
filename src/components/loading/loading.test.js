import React from 'react'
import renderer from 'react-test-renderer'
import Loading from './loading'


test(`Should be loading`,()=>{
  const tree = renderer.create(<Loading/>)
  expect(tree).toMatchSnapshot();
})