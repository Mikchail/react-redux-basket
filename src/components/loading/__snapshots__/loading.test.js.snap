// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Should be loading 1`] = `
<div
  className="loader"
>
  <div
    className="dot"
  />
  <div
    className="dot"
  />
  <div
    className="dot"
  />
  <div
    className="dot"
  />
  <div
    className="dot"
  />
  <div
    className="dot"
  />
  <div
    className="dot"
  />
  <div
    className="dot"
  />
  <div
    className="dot"
  />
</div>
`;
